pom.xml

```diff
@@ -171,6 +172,11 @@
                     <parallel>all</parallel>
                     <useUnlimitedThreads>true</useUnlimitedThreads>
                     <forkCount>1C</forkCount>
+                    <systemPropertyVariables>
+                        <property>
+                            <ui.loading.timeout>${ui.loading.timeout}</ui.loading.timeout>
+                        </property>
```

Comment:
There is no `<property>` tag with `<systemPropertyVariables>`. See [the example](https://maven.apache.org/surefire/maven-surefire-plugin/examples/system-properties.html).